src/bitmessagekivy/baseclass/myaddress.py

```diff
@@ -0,0 +1,207 @@
+from bitmessagekivy.baseclass.common import (
+    avatarImageFirstLetter, AvatarSampleWidget, ThemeClsColor,
+    toast
+)
+from bitmessagekivy.baseclass.popup import MyaddDetailPopup
+from bitmessagekivy.baseclass.myaddress_widgets import HelperMyAddress
+
+
+class ToggleBtn(IRightBodyTouch, MDSwitch):
+    """ToggleBtn class for kivy Ui"""
+
+
+class CustomTwoLineAvatarIconListItem(TwoLineAvatarIconListItem):
+    """CustomTwoLineAvatarIconListItem class for kivy Ui"""
+
+
+class MyAddress(Screen, HelperMyAddress):
+    """MyAddress screen class for kivy Ui"""
+
+    address_label = StringProperty()
+    text_address = StringProperty()
+    addresses_list = ListProperty()
+    has_refreshed = True
+    is_add_created = False
+
+    def __init__(self, *args, **kwargs):
+        """Clock schdule for method Myaddress accounts"""
+        super(MyAddress, self).__init__(*args, **kwargs)
+        Clock.schedule_once(self.init_ui, 0)
+
+    def init_ui(self, dt=0):
+        """Clock schdule for method Myaddress accounts"""
+        self.addresses_list = BMConfigParser().addresses()
+        if state.searcing_text:
+            self.ids.refresh_layout.scroll_y = 1.0
+            filtered_list = [
+                x for x in BMConfigParser().addresses()
+                if self.filter_address(x)
+            ]
+            self.addresses_list = filtered_list
+        self.addresses_list = [obj for obj in reversed(self.addresses_list)]
+        self.ids.tag_label.text = ''
+        if self.addresses_list:
+            self.ids.tag_label.text = 'My Addresses'
+            self.has_refreshed = True
+            self.set_mdList(0, 15)
+            self.ids.refresh_layout.bind(scroll_y=self.check_scroll_y)
+        else:
+            self.ids.ml.add_widget(self.default_label_when_empty())
+            if not state.searcing_text and not self.is_add_created:
+                try:
+                    self.manager.current = 'login'
+                except Exception:
+                    pass
+
+    def set_mdList(self, first_index, last_index):
+        """Creating the mdlist"""
+        data = []
+        for address in self.addresses_list[first_index:last_index]:
+            data.append({
+                'text': BMConfigParser().get(address, 'label'),
+                'secondary_text': address})
+        for item in data:
+            is_enable = BMConfigParser().get(item['secondary_text'], 'enabled')
+            meny = CustomTwoLineAvatarIconListItem(
+                text=item['text'], secondary_text=item['secondary_text'],
+                theme_text_color='Custom' if is_enable == 'true' else 'Primary',
+                text_color=ThemeClsColor,)
+            try:
+                meny.canvas.children[3].rgba = [0, 0, 0, 0] if is_enable == 'true' else [0.5, 0.5, 0.5, 0.5]
+            except Exception:
+                pass
+            meny.add_widget(AvatarSampleWidget(
+                source=state.imageDir + '/text_images/{}.png'.format(
```

Comment:
`os.path.join(state.imageDir, 'text_images', '{}.png'.format(...`

Reply:
> * try adding `__init__.py` in `baseclass` directory

Added `__init__.py` but still pylint giving `no-name-in-module` error.